core/src/main/scala/akka/persistence/jdbc/config/AkkaPersistenceConfig.scala

```diff
@@ -210,7 +210,9 @@ class DurableStateTableColumnNames(config: Config) {
 class DurableStateTableConfiguration(config: Config) {
   private val cfg = config.getConfig("tables.state")
   val refreshInterval: FiniteDuration = config.asFiniteDuration("refreshInterval")
+  val batchSize: Int = config.getInt("batchSize")
   val stopAfterEmptyFetchIterations: Int = config.getInt("stopAfterEmptyFetchIterations")
+  val schemaType: String = config.getString("schemaType")
```

Comment:
Maybe worth failing fast on unsupported rather than later with a match error?

Reply:
I'd be fine with stringly typed here, but if it is not a lot of work actual type here would be better ofc.

Reply:
@johanandren Just one thought .. if I do the check in the config file, even then I will have to repeat that check in `DurableStateQueries` where I instantiate the proper type of `SequenceValUpdater` ..

```scala
val sequenceNextValUpdater = durableStateTableCfg.schemaType match {
  case "H2"       => new H2SequenceNextValUpdater(profile)
  case "POSTGRES" => new PostgresSequenceNextValUpdater(profile)
}
```

But still we can fail fast at the config level at the expense of a double check ..

Reply:
Yeah, I think double check is fine, was thinking mostly about the user experience when accidentally having wrong config and that it would give a good error saying "A and B is supported but not C which you specified"

Reply:
done ..